Formula/llvm.rb

```diff
@@ -109,8 +93,8 @@ class Llvm < Formula
   uses_from_macos "zlib"
 
   patch :p1 do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/c7da61787c62822804dd90192dd9dd254511192c/llvm/10.0.1-llvm.diff"
-    sha256 "86b4e2bf10dfb9a04153e3d4fc744c3f323f3e13fc7b3a7d22e6791d47eb2e2c"
+    url "https://raw.githubusercontent.com/kubkon/formula-patches/2a9895f68da73902d4ce38455068b2ebb84bdc62/llvm/11.0.0-llvm.diff"
```

Comment:
If we have agreed to apply this patch, could you please create a PR to https://github.com/Homebrew/formula-patches (and change URL when it's merged).